From c98d4c780c4a8f6f09073df5a3ee9ab6cdbb2144 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Thu, 13 Apr 2023 11:32:55 +0200
Subject: [PATCH 0161/1141] dt-bindings: spi: stm32: add st,stm32mp25-spi
 compatible

Add st,stm32mp25-spi compatible in dt-bindings.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Iedea1df183b2086754d10b0e82bea34ac0fa5c8a
---
 Documentation/devicetree/bindings/spi/st,stm32-spi.yaml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/Documentation/devicetree/bindings/spi/st,stm32-spi.yaml
+++ b/Documentation/devicetree/bindings/spi/st,stm32-spi.yaml
@@ -38,6 +38,7 @@ properties:
     enum:
       - st,stm32f4-spi
       - st,stm32h7-spi
+      - st,stm32mp25-spi
 
   reg:
     maxItems: 1
@@ -90,7 +91,7 @@ patternProperties:
     properties:
       st,spi-midi-ns:
         description: |
-          Only for STM32H7, (Master Inter-Data Idleness) minimum time
+          Only available since STM32H7, (Master Inter-Data Idleness) minimum time
           delay in nanoseconds inserted between two consecutive data frames.
 
 required:
